pennylane/ops/qubit/qchem_ops.py

```diff
@@ -114,12 +124,39 @@ def compute_matrix(phi):  # pylint: disable=arguments-differ
                 [ 0.0000,  0.2474,  0.9689,  0.0000],
                 [ 0.0000,  0.0000,  0.0000,  1.0000]])
         """
-        c = qml.math.cos(phi / 2)
+        interface = qml.math.get_interface(phi)
```

Comment:
Why this interface requires a different treatment?

Reply:
This is for performance reasons. `qml.stack` typically takes quite some time in the other interfaces, but is very fast with TensorFlow, also see #2759.